### PR TITLE
Feature/component card

### PR DESCRIPTION
**Pull request by weber-cd**

### CODE REVIEW

File: src/components/card/card.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  "& > img": {
+    width: "100%",
+  },
+});
+
+export const cardBodyStyle = getCss({
+  flex: "1 1 auto",
+  minHeight: "1px",
+  padding: CARD_SPACER_X,
+});
```
Comment (SuperJerryshen) [inline src/components/card/card.tsx:40]: These style variables have no need to be exported.

File: src/components/theme/color-mode/default-color-mode.ts [hunk 2]
```diff
@@ -46,6 +48,7 @@ export const DEFAULT_COLOR_MODE: IColorsType = {
   input: BASE_COLORS.gray700,
   inputBorder: BASE_COLORS.gray400,
   placeholder: BASE_COLORS.gray600,
+  borderColor: BASE_COLORS.gray300,
```
Comment (SuperJerryshen) [inline src/components/theme/color-mode/default-color-mode.ts:51]: This color variable has been declared which is `border`.

Comment (weber-cd) [reply src/components/theme/color-mode/default-color-mode.ts]: duplicated variable，delete borderColor，use  `border`.

File: src/components/card/card.tsx (hunk 1 above)
Comment (weber-cd) [reply src/components/card/card.tsx]: unneccessary `export`，deleted

**Pull request merged**